apps/project/tasks.py

```diff
@@ -74,3 +78,333 @@ def regenerate_global_project_assets():
+                tutorial = project.tutorial
+                tutorial_url = Config.ManagerDashboardUrls.tutorial_url(tutorial_id=tutorial_id)
+                return f"Tutorial: <{tutorial_url}|{tutorial}>\n"
+            return "Tutorial: _N/A_\n"
+
+        def status_mapping(status_enum: ProjectStatusEnum):
+            label = status_enum.label
+
+            status_to_icon = {
+                Project.Status.PUBLISHED: ":mega:",
+                Project.Status.PAUSED: ":hand:",
+                Project.Status.ARCHIVED: ":archived:",
+            }
+            return f"{label} {status_to_icon.get(status_enum, '')}".strip()
+
+        def project_type_mapping(project_type_enum: ProjectTypeEnum) -> str:
+            label = project_type_enum.label
+
+            type_to_icon = {
+                Project.Type.FIND: ":find:",
+                Project.Type.COMPARE: ":compare:",
+                Project.Type.VALIDATE: ":validate:",
+                Project.Type.STREET: ":street:",
+                Project.Type.COMPLETENESS: ":completeness:",
+                Project.Type.VALIDATE_IMAGE: ":validate_image:",
+            }
+
+            return f"{label} {type_to_icon.get(project_type_enum, '')}".strip()
+
+        project_url = Config.ManagerDashboardUrls.project_url(project_id=project.id)
+        progress_bar = generate_progress_bar(project.progress)
+        username = format_username(project.created_by)
+        tutorial_text = format_tutorial(project.tutorial_id)
+        status = status_mapping(project.status_enum)
+        project_type = project_type_mapping(project.project_type_enum)
+
+        section_block: dict = {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": (
+                    f"Project Name: <{project_url}|{project.generate_name()}>\n"
+                    f"{tutorial_text}"
+                    f"Project Type: {project_type}\n"
+                    f"Requesting Organization: {project.requesting_organization.name}\n"
+                    f"Status: {status}\n"
+                    f"Created by: {username}\n"
+                    f"Created at: {project.created_at.strftime('%b. %d, %Y, %-I:%M %p')}\n\n"
+                    f"{progress_bar}"
+                ),
+            },
+        }
+
+        if project.image:
+            section_block["accessory"] = {
+                "type": "image",
+                "image_url": get_absolute_uri(project.image.file),
+                "alt_text": "Project Cover Image",
+            }
+
+        return section_block
+
+    @classmethod
+    def get_message_for_project_progress(
+        cls,
+        project: Project,
+    ) -> MapswipeSlack.MapswipeSlackMessageArgumentType:
+        progress = project.progress
+        project_url = Config.ManagerDashboardUrls.project_url(project_id=project.id)
+        website_url = Config.WebsiteKeys.project(firebase_id=project.firebase_id)
+
+        if progress == 100:
```

Comment:
```suggestion
        if progress >= 100:
```

Just to be extra safe.